bool_slice.go

```diff
@@ -0,0 +1,125 @@
+package pflag
+
+import (
+	"strconv"
+	"strings"
+)
+
+// -- boolSlice Value
+type boolSliceValue struct {
+	value   *[]bool
+	changed bool
+}
+
+func newBoolSliceValue(val []bool, p *[]bool) *boolSliceValue {
+	bsv := new(boolSliceValue)
+	bsv.value = p
+	*bsv.value = val
+	return bsv
+}
+
+func (s *boolSliceValue) Set(val string) error {
```

Comment:
Please set a godoc type comment on all exported typed and functions. Even if we failed to do so a bunch in the past, we can stop the bleeding...